src/cmd-buildextend-metal

```diff
@@ -128,7 +128,7 @@ if [ ! -e "$PWD/tmp/ostree-size.json" ]; then
 fi
 size="$(jq '."estimate-mb".final' "$PWD/tmp/ostree-size.json")"
 # 384M is the non-ostree partitions
```

Comment:
Stale comment. Probably better to just reword so as to drop the numbering and mention `src/create_disk.sh`.